requirements/test.txt

```diff
@@ -28,7 +28,7 @@ cffi==1.15.0
     # via cryptography
 charset-normalizer==2.0.7
     # via requests
-cryptography==35.0.0
+cryptography==36.0.0
```

Comment:
Ony one incompatible change mentioned in the [changelog](https://github.com/pyca/cryptography/blob/main/CHANGELOG.rst#3600---2021-11-21) which shouldn't be an issue.